index.js

```diff
@@ -1,16 +1,10 @@
 module.exports = {
   extends: require.resolve('eslint-config-airbnb-base'),
   rules: {
-
     // Disallow comma-dangle
     // http://eslint.org/docs/rules/comma-dangle
     'comma-dangle': [2, 'never'],
```

Comment:
I'm also used to no dangling commas but airbnb has made a good case: https://github.com/airbnb/javascript#commas--dangling

Reply:
I am with you. When we were were working on this style guide I was surprised that some folks felt strongly enough about it to require an override rule. Personally I just don't have a strong opinion about the dangle, and would have been quite happy with leaving airbnb's rule as is

Reply:
I remember vaguely talking about this when we reviewed our eslint config decisions, so it's less surprising to me...IE used to choke on dangling commas; there's no real problem other than preference than that, and that problem isn't relevant anymore, but in my case I still get a little twitchy when I see one, despite otherwise not feeling strongly enough about it to push for one or the other, and it could be other people have the same "trauma" ;-) Though I actually don't remember why we chose to override it, unless it was just that enough people in the room preferred it, for whatever reason.

I have this feeling that if we brought it up for consideration again now it'd get removed. I could be wrong, but if you want to bring it up, I know there's some interest in having the fewest number of overrides acceptable to us collectively.